crates/libs/windows/Cargo.toml

```diff
@@ -13,6 +13,14 @@ readme = "readme.md"
 categories = ["os::windows-apis"]
 exclude = ["features.json"]
 
+[lib]
+# This crate does not contain tests. All tests are in separate crates.
+test = false
+doctest = false
+# This crate does not generate docs because it is too large (rustdoc has problems with very large
+# crates) and because the Windows API is documented elsewhere.
+doc = false
```

Comment:
Similar to the above this crate generates documentation. This is a manual process (as I understand). Unlike the `windows-sys` crate, the documentation here is vital given the numerous transformations the `windows` crate applies to functions. Is documentation generation still supported after this change?

Related: Documentation generation is disabled by default via PR #2676.